lib/wda/xcodebuild.js

```diff
@@ -95,51 +90,25 @@ class XcodeBuild {
       return this.derivedDataPath;
     }
 
-    // try a number of ways to find the derived data folder for this run
-    const pid = await getPidUsingPattern(`xcodebuild.*${this.device.udid}`);
-    if (!pid) {
-      log.debug(`Cannot find xcodebuild's process id, so unable to retrieve DerivedData folder path`);
-      return;
-    }
-    let stdout = '';
+    let output;
     try {
-      const execInfo = await exec('lsof', ['-p', pid]);
-      stdout = execInfo.stdout;
+      const {stdout} = await exec('xcodebuild', ['-project', this.agentPath, '-showBuildSettings']);
+      output = stdout;
```

Comment:
This can just be
```js
let stdout;
try {
  ({stdout} = await exec('xcodebuild', ['-project', this.agentPath, '-showBuildSettings']));
} catch (err) {
```